### ISSUE TITLE
Lipid containing products inhaled

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10216]
**Preferred term label:**
Lipid inhalation

**Synonyms**
breathing in vapors of petroleum jelly, for example after putting it in the nose, or mineral oil laxatives

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Lipoid pneumonia.  https://www.ncbi.nlm.nih.gov/books/NBK554577/ "Exogenous lipoid pneumonia is caused by aspiration or inhalation of lipid-containing products. In the past, the most common agents included mineral oil-based laxatives and nasal decongestants. Other causes include inhalation of petroleum jelly used as a lubricant, i.e., tracheostomy care.... the incidence of exogenous lipoid pneumonia is growing exponentially in all age groups secondary to the phenomena termed "vaping."

**Your nano-attribution (ORCID)**
